src/components/CustomMessage.tsx

```diff
@@ -60,10 +59,9 @@ export default function CustomMessage(props: Props) {
   };
   const { replacementTextList, enableEmojiFeedback, botStudioEditProps } =
     useConstantState();
-  const { stores } = useSendbirdStateContext();
-  const { userId } = useWidgetLocalStorage();
+  const { userId } = useWidgetSession();
```

Comment:
```suggestion
  const { userId: currentUserId } = useWidgetSession();
```

nit but we can combine them :)

Reply:
-> https://github.com/sendbird/chat-ai-widget/pull/238/commits/c00a9aa0df86cb013c1dfdae2fdeb0479e3286cf